### ISSUE TITLE
tslint: true is not a valid option

### ISSUE THREAD

**Issue by johnnyreilly** [TypeStrong/fork-ts-checker-webpack-plugin#15]
Using that results in an error of this nature: 😢 

```
[09:10:53] Problem generating initial assets (js and css) TypeError: Path must be a string. Received true
    at assertPath (path.js:7:11)
    at Object.isAbsolute (path.js:439:5)
    at ForkTsCheckerWebpackPlugin.computeContextPath (C:\source\proverb-react-mobx\node_modules\fork-ts-checker-webpack-plugin\lib\in
dex.js:127:15)
    at ForkTsCheckerWebpackPlugin.apply (C:\source\proverb-react-mobx\node_modules\fork-ts-checker-webpack-plugin\lib\index.js:75:40
```

**Comment by piotr-oles** [comment]
Oops... :/ I will fix it today and write some integration tests :)

**Comment by johnnyreilly** [comment]
Cool!

**Comment by piotr-oles** [comment]
Fixed on v0.2.1 release ;)

**Comment by johnnyreilly** [comment]
Good work!
